iml-system-test-utils/src/vagrant.rs

```diff
@@ -58,12 +58,25 @@ pub async fn up<'a>() -> Result<Command, CmdError> {
     Ok(x)
 }
 
-pub async fn destroy<'a>() -> Result<(), CmdError> {
-    let mut x = vagrant().await?;
+pub async fn destroy<'a>(config: &ClusterConfig) -> Result<(), CmdError> {
+    let mut nodes = config.all();
+    nodes.reverse();
+
+    for node in &nodes {
```

Comment:
What's the benefit to halting in addition to destroying?

Reply:
Will suspend instead of halting.